src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -402,6 +406,10 @@ def _init_seen_tables(self):
     def _sql_alter_to(self, table: Table, target_table_key: str):
         return f"ALTER {table.kind} {escape_sql_identifier(table.key)} SET TBLPROPERTIES ('upgraded_to' = '{target_table_key}');"
 
+    def _sql_add_deprecated_comment(self, table: Table, target_table_key: str):
```

Comment:
Sorry, rereading the link in the docs. The following is probably better:

```suggestion
    def _sql_add_migrated_comment(self, table: Table, target_table_key: str):
```